syntax = "proto3";

package ml_exported_model;

import "orbax/experimental/model/core/protos/saved_model/meta_graph.proto";

option cc_enable_arenas = true;

// SavedModel is the high level serialization format for TensorFlow Models.
// See [todo: doc links, similar to session_bundle] for more information.
message SavedModel {
  // The schema version of the SavedModel instance. Used for versioning when
  // making future changes to the specification/implementation. Initial value
  // at release will be 1.
  int64 saved_model_schema_version = 1;

  // One or more MetaGraphs.
  repeated MetaGraphDef meta_graphs = 2;
}
